packages/client/src/fm-player/components/UsersButton.tsx

```diff
@@ -13,17 +14,31 @@ function UsersButton() {
   }, []);
 
   const handleClick = () => {
-    console.log(currentUsers);
+    setIsExtended(!isExtended);
   };
 
   return (
-    <button
-      type="button"
-      onClick={handleClick}
-      className="bg-green-400 rounded-full p-2 hover:bg-green-600 text-xl font-bold"
-    >
-      <UsersIcon /> {currentUsers.length}
-    </button>
+    <div className="relative">
+      <button
+        type="button"
+        onClick={handleClick}
+        className="absolute bg-green-400 rounded-xl p-3 hover:bg-green-600 text-2xl font-bold leading-none"
+      >
+        <div className="flex">
+          <UsersIcon />
+          <span className="pl-1">{currentUsers.length}</span>
+        </div>
+        {isExtended && (
+        <ul className="text-base font-normal leading-normal pt-1">
+          {currentUsers.map((user, index) => (
+            // eslint-disable-next-line react/no-array-index-key
```

Comment:
we don't have a better key than index at this point, therefore I disabled this check for this case

Reply:
We do have a better a key - the user name itself :D They're unique, that should do.